python/pdo/service_client/storage.py

```diff
@@ -88,11 +88,6 @@ def store_blocks(self, block_data_list, expiration=60) :
             block_ids.append(block_id)
             request_data[block_id] = (block_id, block_data, 'application/octet-stream')
 
-        # for i in range(len(block_data_list)) :
-        #     block_hash = hashlib.sha256(block_data_list[i]).digest()
-        #     block_id = base64.urlsafe_b64encode(block_hash).decode()
-        #     block_ids.append(block_id)
-        #     request_data[block_id] = (block_id, block_data_list[i], 'application/octet-stream')
         request_data['operation'] = json.dumps({'block_ids' : block_ids, 'expiration' : expiration})
         url = "{0}/block/store".format(self.url_base)
         response = requests.post(url, files=request_data, timeout=self.default_timeout)
```

Comment:
So I guess a requests.session is maintained only for communicating with the e-service and not the s-service. IMO, we s-service also benefit from sessions when we replicate the state. We can make this change as part of replication PR.

Reply:
correct. 
we could add a similar facility for the storage service.
not in this commit.